dashboard/src/components/DeploymentForm/DeploymentForm.tsx

```diff
@@ -114,13 +115,12 @@ class DeploymentForm extends React.Component<IDeploymentFormProps, IDeploymentFo
       <div>
         <form className="container padding-b-bigger" onSubmit={this.handleDeploy}>
           {this.props.error && (
-            <DeploymentErrors
-              {...this.props}
-              kubeappsNamespace={kubeappsNamespace}
-              chartName={chartID.split("/")[0]}
-              releaseName={releaseName}
-              repo={chartID.split("/")[1]}
-              version={version.attributes.version}
+            <ErrorSelector
```

Comment:
Ah, I recalled this being an issue when we still had HelmReleases as it would not create the namespace for you, but tiller-proxy will create inexistant namespaces. You're right that this isn't an issue.